src/props/prod-version.props

```diff
@@ -0,0 +1,7 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- If this is a required update, set this to the same version as what we're building.
+	     If this is an optional update, set this to the most recently released prod version. -->
+    <MinimumProdVersion Condition="$(MinimumProdVersion) == ''">1.1.2089.01</MinimumProdVersion>
```

Comment:
It's kind of unfortunate that we'll have to remember to update another version number before releasing... Would it be possible to put this alongside the props we have to update before releasing? Or is there another mechanism we can use to make sure we don't forget this?

Reply:
Agreed that having 2 things in 2 different places is less than ideal. Fortunately, this is a temporary state. After we complete the whole set of features, manual updates of the `release-info.json` files should go away. The files will still need to exist for downlevel clients, but we should be able to completely automate these file updates, just like we already do for the Canary release channel. We currently update 2 things. During the transition (probably just 1 prod release) we'll update 3 things. After the transition, we'll only update 1 thing.